test/Microsoft.AspNetCore.JsonPatch.Test/IntegrationTests/DynamicObjectIntegrationTest.cs

```diff
@@ -287,13 +287,13 @@ public void TestPropertyValue_FromListToNonList_InNestedTypedObject_InDynamicObj
                 IntegerList = new List<int>() { 1, 2, 3 }
             };
 
-            var patchDoc = new JsonPatchDocument();
-            patchDoc.Test("Nested/IntegerList/0", 2);
+            var patchDocument = new JsonPatchDocument();
+            patchDocument.Test("Nested/IntegerList/0", 2);
 
             // Act
             var exception = Assert.Throws<JsonPatchException>(() =>
             {
-                patchDoc.ApplyTo(dynamicTestObject);
+                patchDocument.ApplyTo(dynamicTestObject);
```

Comment:
The dynamic tests seem to have gone wayyy in depth including things like replace and move where a lot of the others don't. I'm not sure if that's good or bad.